### PR TITLE
fix: fix datasets integration tests

### PR DESCRIPTION
**Pull request by bhudnell**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts [hunk 1]
```diff
@@ -101,6 +102,23 @@ describe('datasets create negative tests', () => {
       }
     });
 
+    test('region', async () => {
+      try {
+        const invalidParam: { [id: string]: string } = { ...validLaunchParameters };
+        delete invalidParam.region;
+        await adminSession.resources.datasets.create(invalidParam, false);
```
Comment (htztomic) [inline solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts:109]: add expect.hasAssertions()

File: solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts [hunk 2]
```diff
@@ -101,6 +102,23 @@ describe('datasets create negative tests', () => {
...
+            error: 'Bad Request',
+            message: "requires property 'region'"
+          })
+        );
+      }
+    });
+
     test('all parameters', async () => {
       try {
         await adminSession.resources.datasets.create({}, false);
```
Comment (htztomic) [inline solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts:124]: add expect.hasAssertions()

**Review by htztomic — CHANGES_REQUESTED** [review]

File: solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts (hunk 1 above)
Comment (bhudnell) [reply solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts:109]: included in the `beforeEach` block

File: solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts (hunk 2 above)
Comment (bhudnell) [reply solutions/swb-reference/integration-tests/tests/isolated/datasets/create.test.ts:124]: included in the `beforeEach` block

**Review by htztomic — APPROVED** [review]
Address Tim's comment for deploying and testing integration test locally, otherwise LGTM

**Review by sevu1 — APPROVED** [review]

**Pull request merged**